packages/venia-ui/lib/components/Footer/footer.css

```diff
@@ -1,31 +1,92 @@
 .root {
-    background-color: rgb(var(--venia-grey));
-    border-top: 1px solid rgb(var(--venia-border));
-    color: black;
-    margin-top: 1rem;
-    padding: 0 1rem;
+    border-top: 2px solid hsl(205 004% 096%);
+    color: hsl(205 008% 036%);
+    display: grid;
+    font-size: 14px;
+    grid-template-areas:
+        '. a b .'
+        'c c c c';
+    grid-template-columns: 1fr 640px 384px 1fr;
+    line-height: 1.5;
+    margin: 0 auto;
+    max-width: 1440px;
+    min-height: 15rem;
+    padding-top: 4rem;
+    row-gap: 4rem;
+    width: 100%;
```

Comment:
On mobile view in chrome the footer is being pushed out to what I'm guessing is desktop width:

[![Image from Gyazo](https://i.gyazo.com/b0ee03e8d1833a47a3e60d52ccfa46fd.png)](https://gyazo.com/b0ee03e8d1833a47a3e60d52ccfa46fd)